src/components/InfoSection/InfoSection.js

```diff
@@ -1,26 +1,19 @@
 import Stopwatch from "../Stopwatch/Stopwatch";
-import StyledInfoButton from "../StyledInfoButton/StyledInfoButton";
-import useFailStore from "../../zustand/useFailStore";
-import styled from "styled-components";
 
-const StyledSection = styled.section`
-  display: flex;
-  width: 80vw;
-  gap: 0.4rem;
-  justify-content: center;
-  max-width: 440px;
-`;
+import useFailStore from "../../zustand/useFailStore";
+import StyledDoubleSection from "../StyledDoubleSection/StyledDoubleSection";
```

Comment:
I think you could overcome this double naming when importing by naming the main file `index.js`, so the import looks like this:
```suggestion
import StyledDoubleSection from "../StyledDoubleSection";
```
This is the way we introduce naming when starting with Next.js. However, you would need to rename all component files, so maybe do this when refactoring your code.